README.md

```diff
@@ -91,14 +92,22 @@ Follow the steps [here](https://developer.github.com/apps/building-github-apps/c
 
 ## Running
 
-### Geth
-Launch a local geth node with the bot account unlocked:
+### Ethereum node
+There are two options for connecting to an Ethereum node: either run a local node or connect to a remote one like Infura.
+
+#### Local
+
+In order to launch a local geth node with the bot account unlocked issue the following command:
 
 ```
 #!/bin/bash
 geth --fast --testnet --cache=1024 --datadir=$HOME/.ropsten --verbosity 4 --port 50100 --ipcpath ~/.ropsten/geth.ipc --rpc --rpcaddr 127.0.0.1 --rpcport 8545 --rpcapi db,eth,net,web3,personal --rpccorsdomain "https://wallet.ethereum.org" --unlock "0xYOUR_ADDR" --password <(echo "YOUR_PASSPHRASE")
 ```
 
+#### Remote
+Register at [Infura](https://infura.io/signup). You will receive an email with provider URLs. Paste an URL for the network you're connecting to (Mainnet/Ropsten) into `config.edn` under `:eth-rpc-url` key, and set `:offline-signing?` to true.
```

Comment:
Are there any potential pitfalls around connecting to mainnet? Situations where a user could lose funds or similar?

Reply:
There could be I believe. Do you think reference to Mainnet should be deleted?

Reply:
Maybe. Or maybe have a little warning like this (including the quote thingy):

> :warning: Be careful not to use the Mainnet unless you want to test with actual ETH tokens.

"actual ETH tokens" is probably not the right way to phrase things here but I guess you get what I mean?

Reply:
I think it's better to remove the mention of Mainnet altogether. @andytudhope - what do you think?

Reply:
Go for it. We can add it back later. :)

Reply:
Maybe also mention that Ropsten is a network explicitly for testing / development stuff.

Reply:
Done.